package.json

```diff
@@ -21,8 +21,8 @@
     "verify": "npm-run-all build lint test"
   },
   "dependencies": {
-    "@patternfly/react-core": "^5.1.0",
-    "@patternfly/react-table": "^5.1.0",
+    "@patternfly/react-core": "^5.4.2",
+    "@patternfly/react-table": "^5.4.8",
```

Comment:
These updates are needed to get the most up-to-date props for the new Wizard component